docs/quick-start.md

```diff
@@ -30,15 +30,15 @@ We use the `--packages` argument to download the graphframes package and any dep
 <div data-lang="scala"  markdown="1">
 
 {% highlight bash %}
-$ ./bin/spark-shell --packages graphframes:graphframes:0.5.0-spark2.1-s_2.11
+$ ./bin/spark-shell --packages graphframes:graphframes:0.6.0-spark2.1-s_2.11
```

Comment:
release notes say `2.3` support the version here seems off.

Reply:
good catch.